file-formats/doma/examples/z_aff_example_doma.doma.json

```diff
@@ -0,0 +1,24 @@
+{
+    "formatVersion": "1",
+    "header": {
+      "description": "Example DOMA for ABAP file formats",
+      "originalLanguage": "en"
+    },
+    "negative_values": false,
+    "case_sensitive": false,
+    "data_type": "CHAR",
+    "length": 3,
+    "decimals": 2,
+    "output_length": 3,
+    "conversion_routine": "",
+    "value_table": "",
+    "fixed_values": {
+        "fixed_value": "D",
+        "description": "Test fixed values description"
+    },
+    "fixed_value_intervals": {
+        "low_limit": "1",
+        "high_limit": "5",
+        "description": "Test interval description"
+    }
+  }
```

Comment:
You might copy&paste the correct formatting from here
```
{
  "formatVersion": "1",
  "header": {
    "description": "Example DOMA for ABAP file formats",
    "originalLanguage": "en"
  },
  "negative_values": false,
  "case_sensitive": false,
  "data_type": "CHAR",
  "length": 3,
  "decimals": 2,
  "output_length": 3,
  "conversion_routine": "",
  "value_table": "",
  "fixed_values": {
    "fixed_value": "D",
    "description": "Test fixed values description"
  },
  "fixed_value_intervals": {
    "low_limit": "1",
    "high_limit": "5",
    "description": "Test interval description"
  }
}
```

Reply:
with an empty line at the end 😄

Reply:
Should be fixed now. Thank you.